desc/equilibrium/coords.py

```diff
@@ -504,3 +503,65 @@ def to_sfl(
     eq_sfl.L_lmn = L_lmn_sfl
 
     return eq_sfl
+
+
+def rtz_grid(eq, radial, poloidal, toroidal, coordinates, period, jitable=True):
+    """Return DESC coordinate grid from given coordinates.
+
+    Create a meshgrid from the given coordinates, and return the
+    paired DESC coordinate grid.
+
+    Parameters
+    ----------
+    eq : Equilibrium
+        Equilibrium on which to perform coordinate mapping.
+    radial : ndarray
+        Sorted unique radial coordinates.
+    poloidal : ndarray
+        Sorted unique poloidal coordinates.
+    toroidal : ndarray
+        Sorted unique toroidal coordinates.
+    coordinates : str
+        Input coordinates that are specified by the arguments, respectively.
+        raz : rho, alpha, zeta
+        rpz : rho, theta_PEST, zeta
+        rtz : rho, theta, zeta
+    period : tuple of float
```

Comment:
alpha is only 2pi periodic in theta_PEST not zeta. How does the user know which one you are talking about?
What does period in inbasis mean? In basis can have both theta and zeta.

Reply:
I just copied the docstring from the `def map_coordinates` method in this file. I think it's more like this. 
```python
# want to make grid with three (long) field lines on each flux surface
eq = get("W7-X")
rho = [0, 0.5, 1]
alpha = [0, pi, 3pi/2]
zeta = np.linspace(0, 100000000, 1000000000).
grid = rtz_grid(eq, rho, alpha, zeta, coordinates="raz", period=(np.inf, 2 * np.pi, np.inf))
```
By specifying 2pi, you are telling the code that `alpha % (2pi) == alpha`.